modules/nf-core/picard/positionbaseddownsamplesam/main.nf

```diff
@@ -0,0 +1,62 @@
+process PICARD_POSITIONBASEDDOWNSAMPLESAM {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/picard:3.1.1--hdfd78af_0' :
+        'biocontainers/picard:3.1.1--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(bam), val(fraction)
+
+    output:
+    tuple val(meta), path("*.ds*.bam")        , emit: bam
+    tuple val(meta), path("*.ds*.bai")        , emit: bai, optional:true
+    tuple val(meta), env(ACTUAL_NUM_READS)    , emit: num_reads, optional:true
+    path  "versions.yml"                      , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def avail_mem = 3
+    if (!task.memory) {
+        log.info '[Picard PositionBasedDownsampleSam] Available memory not known - defaulting to 3GB. Specify process memory requirements to change this.'
```

Comment:
```suggestion
        log.info '[Picard PositionBasedDownsampleSam] Available memory not known - defaulting to 3GB. Specify process memory requirements to change this.'
```
Extra points if you can give an example on how to specify them

Reply:
@mashehu , what exactly were you thinking to put here? This is a relatively standard template that is in most of the gatk and picard tools.